modules/subscriptions.php

```diff
@@ -75,6 +77,7 @@ function __construct() {
 		add_action( 'post_submitbox_misc_actions', array( $this, 'subscription_post_page_metabox' ) );
 
 		add_action( 'transition_post_status', array( $this, 'maybe_send_subscription_email' ), 10, 3 );
+		add_action( 'wp_insert_post', array( $this, 'maybe_trigger_send_email_action' ), 11, 3 );
```

Comment:
Could we fire `jetpack_email_post_to_subscribers` from within `maybe_send_subscription_email`?